src/pages/Checkout/Checkout.js

```diff
@@ -0,0 +1,69 @@
+import { useLocation, useNavigate } from "react-router-dom";
+
+import { Header } from "../../components";
+
+import styles from "./Checkout.module.css";
+
+const formatDate = (dateParam) => {
+  const date = new Date(dateParam);
+  const day = date.getDate();
+  const month = date.getMonth() + 1;
+  const year = date.getFullYear();
+
+  return `${day < 10 ? '0' + day : day}/${month < 10 ? '0' + month : month}/${year}`;
+};
+
+export default function Checkout() {
+  const { state: menuData } = useLocation();
+  const navigate = useNavigate();
+
+  const handleConfirm = () => {
+    alert('Confirmando');
+    // TODO: BACK
+    navigate(-1)
+  }
+
+  return (
+    <div className={styles.checkout}>
+      <Header />
+      <main>
+        <h1>Confirmar Reserva</h1>
+        <div className={styles.checkout__available}>
+          <h2>Saldo disponível</h2>
+          <span>
+            R$ <span id={styles.checkout__value}>30.00</span>
+          </span>
+        </div>
+        <div className={styles.checkout__filters}>
+          <span id={styles.checkout__date}>
+            {formatDate(menuData.date)}
```

Comment:
Essa função não é necessária, a biblioteca `date-fns` já tem métodos pra isso
```suggestion
        import {format} from 'date-fns';
        [...]
        {format(menuData.date, 'dd/MM/yyyy')}
```